pkg/cnb/cnb_metadata.go

```diff
@@ -57,23 +57,23 @@ func (r *RemoteMetadataRetriever) GetBuilderImage(repo registry.ImageRef) (Build
 
 	return BuilderImage{
 		BuilderBuildpackMetadata: metadata.Buildpacks,
-		Identifier:               identifier.String(),
+		Identifier:               identifier,
```

Comment:
should this be Image too?

Reply:
That is an actual built image identifier. Not the same thing as an image because it always includes the digest